cmd/eelbot/main.go

```diff
@@ -84,3 +127,33 @@ func main() {
 		panic(err)
 	}
 }
+
+// Converts a map[any]any to a map[string]any.
+func toStrKeys(a any) any {
```

Comment:
I'm tempted to say only support .json lol

Reply:
Yeah, I thought of that too. It's not too bad right now. We can drop yaml support if it becomes too much of a pain down the line.